src/pages/CreatePoll.js

```diff
@@ -375,146 +635,64 @@ class CreatePoll extends Component {
-                },
-                {
-                  title: 'Vote Options',
-                  placeholder: 'Add possible voting options',
-                  value: option,
-                  onChange: e => this.handlePollState(e, 'option'),
-                  error: choicesError,
-                  failureMessage:
-                    choicesError && 'Must be at least two voting options',
-                  after: (
-                    <Button
-                      css={{ alignSelf: 'center', marginLeft: '10px' }}
-                      width="190px"
-                      onClick={this.addPollVoteOption}
-                    >
-                      Add Option
-                    </Button>
-                  )
-                }
-              ].map((args, i) => (
-                <CreatePollInput key={i} {...args} />
-              ))}
-
-              <SectionWrapper>
-                <StyledBody />
-                <VoteOptionsGrid>
-                  {choices.map((opt, idx) => (
-                    <Card
-                      key={idx}
-                      css={{
-                        display: 'flex',
-                        flexDirection: 'row',
-                        justifyContent: 'space-between',
-                        alignItems: 'center',
-                        width: '300px',
-                        padding: '10px'
-                      }}
-                    >
-                      <OptionText>
-                        #{idx} - {opt}
-                      </OptionText>
-                      {idx > 0 && (
-                        <CloseIcon onClick={() => this.removePollOption(idx)} />
-                      )}
-                    </Card>
-                  ))}
-                </VoteOptionsGrid>
-              </SectionWrapper>
-
-              <CreatePollTime
-                {...{
-                  start,
-                  end,
-                  timeError,
-                  handleParentState
-                }}
-              />
-
-              <SectionWrapper>
-                <StyledBody>Proposal:</StyledBody>
-                <Box width="600px">
-                  <ReactMde
-                    value={content}
-                    onChange={value => this.setState({ content: value })}
-                    selectedTab={selectedTab}
-                    onTabChange={tab => this.setState({ selectedTab: tab })}
-                    generateMarkdownPreview={markdown =>
-                      Promise.resolve(converter.makeHtml(markdown))
-                    }
+          <Stepper
+            css={{ paddingBottom: '30px' }}
+            selected={step}
+            steps={[
+              'Create Poll Markdown',
+              'Deploy Poll Contract',
+              'Poll Result'
+            ]}
+          />
+          {(() => {
+            switch (step) {
+              case 0:
+                return (
+                  <CreatePollMarkdown
+                    {...{
```

Comment:
in cases like these, it might be a little clearer to pass the props in directly (w/o spreading them) like 
```
<CreatePollMarkdown
   state={this.state}
   handlePollState={this.handlePollState}
```